connector/src/main/java/org/geysermc/connector/network/translators/java/world/JavaBlockValueTranslator.java

```diff
@@ -53,5 +52,36 @@ public void translate(ServerBlockValuePacket packet, GeyserSession session) {
             blockEventPacket.setEventType(1);
             session.getUpstream().sendPacket(blockEventPacket);
         }
+        if (packet.getValue() instanceof NoteBlockValue) {
+            NoteBlockValueType type = (NoteBlockValueType) packet.getType();
+
+            blockEventPacket.setEventType(type.ordinal());
+
+            session.getUpstream().sendPacket(blockEventPacket);
```

Comment:
Looks like you can just put this at the end of the file since it's guaranteed to be sent anyway.